docker-compose.yml

```diff
@@ -36,11 +37,12 @@ services:
             - BEHAVIOR_TCHCLIENT=client,server,encoding
             - BEHAVIOR_TCHSERVER=client,server,encoding
             # BEHAVIOR_HTTPCLIENT TODO
-            # BEHAVIOR_HTTPSERVER TODO
+            - BEHAVIOR_HTTPSERVER=client,httpserver
```

Comment:
I used the confusing existing convention. This tests yarpc-client with http transport against an http server. I have mentioned few time how our tests are confusingly named, I intend to go through a renaming pass at some point.